input/ig.xml

```diff
@@ -558,39 +558,39 @@
 				<reference value="Patient/ronny-irvine"/>
 			</reference>
 			<name value="Patient - Mr. Ronny Irvine (missing data for birth date)"/>
-			<description value="Mr. Ronny Irvine with missing data for birth date"/>
+			<description value="Shows a patient with missing data for birth date example for *AU Core Patient* profile."/>
```

Comment:
To be consistent with the MedicationRequest suggestion I made. This should be:

Shows an example of a Torres Strait Islander patient with DVA number for *AU Core Patient* profile illustrating missing data for birth date.